src/core/ext/filters/client_channel/xds/xds_api.cc

```diff
@@ -260,7 +273,7 @@ envoy_api_v2_DiscoveryRequest* CreateDiscoveryRequest(
   if (build_version != nullptr) {
     envoy_api_v2_core_Node* node_msg =
         envoy_api_v2_DiscoveryRequest_mutable_node(request, arena);
-    PopulateNode(arena, node, build_version, node_msg);
+    PopulateNode(arena, node, build_version, "", node_msg);
```

Comment:
Why "PROXYLESS_CLIENT_HOSTNAME" is set to empty string for xDS requests? Does it matter? Can it be the same `Node` as used for LRS? Or can the `Node` for xDS requests not have that metadata field?

Reply:
This metadata field should not be present in xDS requests (which is what this code does -- `PopulateNode()` will only populate this field if the passed-in string is non-empty).

This metadata field is needed only for LRS.